code-challenges/sort/challenges-03.test.js

```diff
@@ -0,0 +1,285 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1
+
+Write a function named sortBackwards that takes in an array of numbers and returns the same array, with the numbers sorted, highest to smallest.
+------------------------------------------------------------------------------------------------ */
+
+const sortBackwards = (arr) => {
+  // Solution code here...
+  arr.sort( (a,b) => (b-a));
```

Comment:
On these challenges where you are altering an array on one line and then returning it, you can try combining them into one line:

return arr.sort(  (a,b) => (b-a));

You don't have to do this though!!!